--- /etc/xdg/autostart/gnome-keyring-gpg.desktop	2012-05-20 16:48:23.226280183 +1000
+++ /etc/xdg/autostart/gnome-keyring-gpg.desktop	2012-05-20 16:49:44.328434899 +1000
@@ -3,7 +3,7 @@ Type=Application
 Name=GPG Password Agent
 Comment=GNOME Keyring: GPG Agent
 Exec= gnome-keyring-daemon --start --components=gpg
-OnlyShowIn=GNOME;Unity;
+OnlyShowIn=GNOME;Unity;LXDE;Lubuntu;
 X-GNOME-Autostart-Phase=Initialization
 X-GNOME-AutoRestart=false
 X-GNOME-Autostart-Notify=true
