--- /etc/xdg/autostart/gnome-keyring-secrets.desktop	2012-05-20 16:48:23.226280183 +1000
+++ /etc/xdg/autostart/gnome-keyring-secrets.desktop	2012-05-20 16:49:40.712338820 +1000
@@ -3,7 +3,7 @@ Type=Application
 Name=Secret Storage Service
 Comment=GNOME Keyring: Secret Service
 Exec=/usr/bin/gnome-keyring-daemon --start --components=secrets
-OnlyShowIn=GNOME;Unity;
+OnlyShowIn=GNOME;Unity;LXDE;Lubuntu;
 X-GNOME-Autostart-Phase=Initialization
 X-GNOME-AutoRestart=false
 X-GNOME-Autostart-Notify=true
